cms/templates/js/timed-examination-preference-editor.underscore

```diff
@@ -26,6 +26,17 @@
         </div>
         <ul class="list-fields list-input">
             <li class="field field-checkbox checkbox-cosmetic">
+                <input type="checkbox" id="id_practice_exam" name="practice_exam" class="input input-checkbox" />
+                <label for="id_practice_exam" class="label">
+                  <i class="icon fa fa-check-square-o input-checkbox-checked"></i>
+                  <i class="icon fa fa-square-o input-checkbox-unchecked"></i>
+                  <%- gettext('This exam is for practice only') %>
+                </label>
+            </li>
+            <p class='field-message'> <%- gettext('Students will take this exam as a practice exam.') %> </p>
```

Comment:
"Learners can experience the proctoring software setup process and try some example problems. This exam is for practice only and does not affect grading or credit eligibility."